packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -716,6 +732,7 @@ abstract class ResidentRunner {
       printStatus('You can dump the widget hierarchy of the app (debugDumpApp) by pressing "w".');
       printStatus('To dump the rendering tree of the app (debugDumpRenderTree), press "t".');
       printStatus('For layers (debugDumpLayerTree), use "L"; accessibility (debugDumpSemantics), "S".');
+      printStatus('To toggle the widget inspector (widgetInspector), press "i".');
```

Comment:
The parenthetical here is supposed to be the thing you can look up in the docs to learn about it, and also the thing that you can toggle from code to do the same thing. So in this case, it should probably be `WidgetsApp.showWidgetInspectorOverride`.